Modules/Core/Common/CMake/itkCheckHasFenvtStructMember.cxx

```diff
@@ -21,15 +21,14 @@
 int
 main()
 {
-  fenv_t fenv;
+  [[maybe_unused]] fenv_t fenv;
 #if defined(ITK_CHECK_FENV_T_CONTROL)
-  (void)sizeof(fenv.__control);
+  [[maybe_unused]] const auto tempSize = sizeof(fenv.__control);
 #elif defined(ITK_CHECK_FENV_T_CONTROL_WORD)
-  (void)sizeof(fenv.__control_word);
+  [[maybe_unused]] const auto tempSize = sizeof(fenv.__control_word);
 #elif defined(ITK_CHECK_FENV_T_CW)
-  (void)sizeof(fenv.__cw);
+  [[maybe_unused]] const auto tempSize = sizeof(fenv.__cw);
 #else
-  (void)fenv;
 #  error \
     "Unknown fenv_t struct member test: Make sure to specify a compile definition of the form -DITK_CHECK_FENV_T_xxx"
 #endif
```

Comment:
It seems unnecessarily to me to prevent a warning here, as it is directly followed by a compiler error anyway! So it won't solve any CI failures. Anyway, no problem.